src/pages/PreviewScreen.jsx

```diff
@@ -3,13 +3,26 @@ import { useHistory, useParams, useLocation } from "react-router-dom";
 import { Preview } from "@100mslive/hms-video-react";
 import { AppContext } from "../store/AppContext";
 import getToken from "../services/tokenService";
+import { convertLoginInfoToJoinConfig } from "../store/appContextUtils";
+import { useState } from "react";
+import { useEffect } from "react";
+import { Notifications } from "../views/components/notifications";
 
 const PreviewScreen = () => {
   const history = useHistory();
   const context = useContext(AppContext);
   const { loginInfo, setLoginInfo, setMaxTileCount, tokenEndpoint } = context;
   const { roomId: urlRoomId, role: userRole } = useParams();
   const location = useLocation();
+  const [token, setToken] = useState("");
+
+  useEffect(() => {
+    getToken(tokenEndpoint, loginInfo.env, "a", userRole, urlRoomId).then(
```

Comment:
can you add a comment here why "a" is necessary.

Reply:
This was causing the token generation service to return `null` for the token. Sending any non-empty string yields the token.

Reply:
and maybe we should change "a" to sth like "dummy" too, to convey the signficance.

Reply:
yes I know of the reason. It's more for future readers of the codebase.

Reply:
code answers what and comments answer why.